tests/test_salesforce_oauth_request.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import salesforce_oauth_request
+
+USERNAME = None
+PASSWORD = None
+SECURITY_TOKEN = None
+CONSUMER_KEY = None
+CONSUMER_SECRET = None
+
+
+def setup_module(module):
+    global USERNAME, PASSWORD, SECURITY_TOKEN, CONSUMER_KEY, CONSUMER_SECRET
```

Comment:
you could make it a dictionary to avoid all this copy/paste. then grabbing from the environment vars and checking they're all non-empty would be just loops.